scripts/install

```diff
@@ -193,13 +200,15 @@ ynh_script_progression --message="Configuring a systemd service..."
 
 # Create a dedicated systemd config
 ynh_add_systemd_config --service=$service_name
+ynh_add_systemd_config --service=captiveportal_fakedns --template=captiveportal_fakedns
 
 #=================================================
 # INTEGRATE SERVICE IN YUNOHOST
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..."
 
 yunohost service add $service_name --description "Creates a Wi-Fi access point" --test_status "systemctl is-active hostapd"
+yunohost service add captiveportal_fakedns --description "Captive portal dns service" --test_status "systemctl is-active captiveportal_fakedns"
```

Comment:
The `test_status` stuff is probably useless here, it's testing the `is-active` status on the same service name ... The previous instruction does have a legit `--test_status` because it's checking the status of `hostapd` instead of `$service_name`